src/Riok.Mapperly/Symbols/MethodAccessorMember.cs

```diff
@@ -0,0 +1,46 @@
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using static Riok.Mapperly.Emit.Syntax.SyntaxFactoryHelper;
+
+namespace Riok.Mapperly.Symbols;
+
+public class MethodAccessorMember : IMappableMember
+{
+    private readonly IMappableMember _mappableMember;
+    private readonly string _methodName;
+    private readonly bool _isMethod;
```

Comment:
I think the isMethod name here is still confusing as this is inside a `MethodAccessorMember`. Probably `targetValueIsMethodParameter` or similar works better here?